### PR TITLE
JBuilders for payment and payment type

### PR DESCRIPTION
**Pull request by awadyas**
**NOTE: DO NOT discuss internal CommitChange information in your PR; this PR will be public.
Link back to the issue in the Tix repo when you need to do that.**

### Issue
First step of https://github.com/CommitChange/tix/issues/3944

### Deployment plan
Merge with supporter_level_goal any time

### CODE REVIEW

File: app/views/api_new/common/_payment_type.json.jbuilder [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+
+json.payment_type "offline_payment | stripe"
+json.check_no "12345678 | nil"
```
Comment (awadyas) [inline app/views/api_new/common/_payment_type.json.jbuilder:7]: Still waiting on answers for where to get this info.

Comment (awadyas) [reply app/views/api_new/common/_payment_type.json.jbuilder:7]: https://github.com/CommitChange/tix/issues/3942#issuecomment-1468382005

File: app/views/api_new/supporters/_supporter.json.jbuilder [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
 
-json.(supporter, :name, :organization, :phone, :anonymous, :deleted)
+json.(supporter, :name, :email, :organization, :phone, :anonymous, :deleted)
```
Comment (wwahammy) [inline app/views/api_new/supporters/_supporter.json.jbuilder]: Can you make this commit a separate PR as well as add tests for any place this is currently used?

**Comment by awadyas** [comment]
Closing bc we have abandoned `payments.created` in favor of using offline transaction charges and stripe transactions to get the data we need
